src/components/Sidebar/Switcher/Switcher.js

```diff
@@ -0,0 +1,28 @@
+import React, {useState} from 'react';
```

Comment:
Codacy has a fix for the issue: [A space is required after '{'.](https://app.codacy.com/gh/alxshelepenok/gatsby-starter-lumen/pullRequest?prid=7771711)
```suggestion
import React, { useState} from 'react';
```